### ISSUE TITLE
Ring Alarm Not Responding to Arming Commands

### ISSUE THREAD

**Issue by bucks7756** [asishrs/smartthings-ringalarmv2#51]
So I have been able to connect my Ring account and all my devices come back to SmartThings.  However, when I try to arm through webcore or the smartthings ring app, I get a notification from smartthings that the alarm system is armed however the Ring Alarm never actually arms.  Any ideas?  Thanks.

**Comment by asishrs** [comment]
do you have any door locks connected to your Ring Account?

**Comment by jhstroebel** [comment]
asishrs, I do... and have this issue as well.  I had re-written your original code to include lock control, and attempted to make the same modifications to the new code to remedy the issue, but no luck.  Seems like you might have a clue why this doesn't work?  I went down this rabbit hole and figured it out before, but if you have a clue to start me down my search, it'd be much appreciated!  I forget the vast majority of what I learned last time and one of these days I'll learn documentation is my friend...
